packages/cozy-mespapiers-lib/src/components/CompositeHeader/CompositeHeaderImage.spec.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import CompositeHeaderImage from './CompositeHeaderImage'
+
+const setup = ({ icon, fallbackIcon } = {}) => {
+  return render(
+    <CompositeHeaderImage
+      icon={icon}
+      fallbackIcon={fallbackIcon}
+      iconSize={'medium'}
+      title="Title"
+      text="Text"
+    />
+  )
+}
+
+describe('CompositeHeaderImage', () => {
+  it('should return nothing if icon & fallbackIcon props is undefined', () => {
+    const { container } = setup()
+
+    expect(container).toMatchInlineSnapshot('<div />')
```

Comment:
nit: don't we use https://github.com/testing-library/jest-dom ?

to be honest, that test would be better with a https://github.com/testing-library/jest-dom#tobeemptydomelement=
```suggestion
    expect(container).toBeEmptyDOMElement()
```

Reply:
En effet, merci j'étais passé à côté 👍